examples/chat/client/tasks.nim

```diff
@@ -356,12 +480,28 @@ proc stopWakuChat2*() {.task(kind=no_rts, stoppable=false).} =
   trace "task sent event to host", event=eventEnc, task
   asyncSpawn chanSendToHost.send(eventEnc.safe)
 
+proc lightpushHandler(response: PushResponse) {.gcsafe.} =
+  trace "received lightpush response", response
```

Comment:
Is `lightpush` meant to facilitate notifications?

Reply:
If so, we could send an event to the host, even if we don't do anything with it right now (again likely out of scope of this PR, but just an idea).

Reply:
`lightpush` is as, afaik, supposed to be a mechanism for message publication that is less resource intensive than the publication via the usual "relay node" mechanism. In this case, I've simply ported nim-waku's `chat2.nim` code without deep consideration of what's involved if lightpush is active (and nim-waku's `chat2.nim` code for this case seems a bit under developed / stub'ed in, but I made a place for it to cover the bases).

Reply:
Very interesting. Could we possibly use light push without running a relay node? Maybe that could be a cli option when starting up the example chat client. Something to explore another time...